src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -2178,19 +2178,20 @@ public void BuildReleaseApplicationWithNugetPackages ()
 		{
 			var proj = new XamarinAndroidApplicationProject () {
 				IsRelease = true,
-				Packages = {
+				PackageReferences = {
 					KnownPackages.AndroidSupportV4_21_0_3_0,
 				},
 			};
 			using (var b = CreateApkBuilder (Path.Combine ("temp", TestContext.CurrentContext.Test.Name))) {
 				Assert.IsTrue (b.Build (proj), "Build should have succeeded.");
-				DirectoryAssert.Exists (Path.Combine (Root, "temp","packages", "Xamarin.Android.Support.v4.21.0.3.0"),
-										"Nuget Package Xamarin.Android.Support.v4.21.0.3.0 should have been restored.");
+				var assets = b.Output.GetIntermediaryAsText (Path.Combine ("..", "project.assets.json"));
+				StringAssert.Contains ("Xamarin.Android.Support.v4", assets,
+					"Nuget Package Xamarin.Android.Support.v4.21.0.3.0 should have been restored.");
 			}
 		}
 
 		[Test]
-		public void BuildWithResolveAssembliesFailure ([Values (true, false)] bool usePackageReference)
+		public void BuildWithResolveAssembliesFailure ()
```

Comment:
Yeah as @jonpryor said, we should keep a couple tests that still use `packages.config`, maybe they can be parameterized just like this one?